tests/library/modernizr.spec.ts

```diff
@@ -88,13 +91,16 @@ it('Safari Desktop', async ({ browser, browserName, platform, server, headless }
   expect(actual).toEqual(expected);
 });
 
-it('Mobile Safari', async ({ playwright, browser, browserName, platform, server, headless }) => {
+it('Mobile Safari', async ({ playwright, browser, browserName, platform, httpsServer, headless }) => {
```

Comment:
https is required here in order to have a Secure Context.

Reply:
Unlucky!